extension/storage/filestorage/config_test.go

```diff
@@ -95,3 +95,76 @@ func TestHandleProvidingFilePathAsDirWithAnError(t *testing.T) {
 	require.Error(t, err)
 	require.EqualError(t, err, file.Name()+" is not a directory")
 }
+
+func TestCompactionDirectory(t *testing.T) {
+	tests := []struct {
+		name     string
+		validate func(*testing.T)
+	}{
+		{
+			name: "directory-must-exists-error",
+			validate: func(t *testing.T) {
+				f := NewFactory()
```

Comment:
Most of the validate function body is the same for every test, do you think it is worth to refactor it so that tests is a `[]struct {name string; err error }` or similar?

Reply:
Thanks, I've refactored them, but we still need to call `CreateDefaultConfig()` since some of the fields, while not relevant to this test, are still required.